web/src/components/organisms/Project/Schema/index.tsx

```diff
@@ -55,12 +49,7 @@ const ProjectSchema: React.FC = () => {
     <>
       <PaddedContent>
         <SchemaStyledMenu>
-          <SchemaMenu
-            selectModel={selectModel}
-            defaultSelectedKeys={[model?.id ?? ""]}
-            models={models}
-            handleModalOpen={handleModelModalOpen}
-          />
+          <ModelsMenu title={t("Models")} selectModel={selectModel} />
```

Comment:
Similar as in another component the ModelsMenu is wrapped in a SchemaStyledMenu. Either rename it to Wrapper or SchemaMenuWrapper or something OR remove it and make a StyledModelsMenu and add the styles directly to the menu. Whichever makes the most sense

Reply:
Also fixed as the above one, removed the wrapper, and used styled component.